server/grapevine-api/middlewares/validateRequest.coffee

```diff
@@ -1,33 +1,26 @@
 jwt = require 'jwt-simple'
-# retrieveUser = require('../routes/users').retrieveUser
+retrieveUser = require('../routes/auth').retrieveUser
 
```

Comment:
The auth routes handles login and registration. I just wanted to use the `retrieveUser` method defined in this module so I wouldn't duplicate code, but I agree that the whole this is not organized as nicely as it could be.